modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapLink.vue

```diff
@@ -88,10 +77,13 @@ export default {
       target: null,
       active: false,
       hasLinkOnOpen: false,
+      // uses key hack to force refresh among selections
+      // so that schema values aren't erronously carried from selection to selection
+      schemaKey: 'apostrophe3whips',
```

Comment:
I think you can literally call this `lastSelectionTime`. It conveys what we're doing: treating the current selection as a piece of state that potentially requires a refresh when changed.

Reply:
🕐